assets/_scss/elements/_typography.scss

```diff
@@ -17,12 +17,22 @@ p {
 }
 
 a {
-  color: $color-base-link;
-  &:hover, &:active, &:visited {
-    color: $color-base-link;
+  border-bottom: 1px dotted;
+  &:hover {
+    border-bottom: 1px solid;
   }
 }
 
+.usa-link {
```

Comment:
Isn't this technically more complex then an element because it uses a class? It kinda means that it takes the base element (all the rules of the a tag) and builds on top of them.